### PR TITLE
GEOT-5329: Fix for timezone related date shifting

### PR DESCRIPTION
**Pull request by awaterme**
Fix for [GEOT-5329](https://osgeo-org.atlassian.net/browse/GEOT-5329)

### CODE REVIEW

File: modules/extension/xsd/xsd-core/src/test/java/org/geotools/xml/DateConversionTimezoneTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2007-2015, Open Source Geospatial Foundation (OSGeo)
```
Comment (aaime) [inline modules/extension/xsd/xsd-core/src/test/java/org/geotools/xml/DateConversionTimezoneTest.java]: Only 2015

File: modules/extension/xsd/xsd-core/src/main/java/org/geotools/xs/bindings/XSDateBinding.java [hunk 2]
```diff
@@ -21,6 +21,7 @@
 import java.util.TimeZone;
```
Comment (aaime) [inline modules/extension/xsd/xsd-core/src/main/java/org/geotools/xs/bindings/XSDateBinding.java]: Copyright header should be updated to "2002-2015"

File: modules/extension/xsd/xsd-core/src/main/java/org/geotools/xml/XmlConverterFactory.java [hunk 3]
```diff
@@ -160,10 +160,15 @@ private String convertToString(Object unconvertedValue) {
             } else if (unconvertedValue instanceof java.util.Date) {
```
Comment (aaime) [inline modules/extension/xsd/xsd-core/src/main/java/org/geotools/xml/XmlConverterFactory.java]: Copyright header should be updated to "2002-2015"

File: modules/extension/xsd/xsd-core/src/test/java/org/geotools/xml/DateConversionTimezoneTest.java [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+/**
+ * Tests for timezone aspects of date conversion in {@link XmlConverterFactory} and {@link XSDateBinding}.
+ * 
+ * @author awaterme
+ *
+ */
+public class DateConversionTimezoneTest extends TestCase {
+
+    private XmlConverterFactory sut1 = new XmlConverterFactory();
```
Comment (aaime) [inline modules/extension/xsd/xsd-core/src/test/java/org/geotools/xml/DateConversionTimezoneTest.java]: "sut" stands for... what?

Comment (awaterme) [inline modules/extension/xsd/xsd-core/src/test/java/org/geotools/xml/DateConversionTimezoneTest.java]: System Under Test, https://en.wikipedia.org/wiki/System_under_test

**Comment by jodygarnett** [comment]
Any progress here? Feature freeze is next week...

**Comment by awaterme** [comment]
Hi,

I did not get answers to my last questions on the mailing list, so I did not continue.
Currently I am busy but I like to enhance the proposal in the future.
Could you leave it open please?

**Comment by awaterme** [comment]
Created a new PR [#1116](https://github.com/geotools/geotools/pull/1116) in order to have one commit only.

**Pull request closed without merging**